modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,235 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-running-dpdk-rootless-tap_{context}"]
+= Using the TAP CNI to run a rootless DPDK workload with kernel access
+
+DPDK applications can use `virtio-user` as an exception path to inject certain types of packets, such as log messages, into the kernel for processing. For more information about this feature, see link:https://doc.dpdk.org/guides/howto/virtio_user_as_exception_path.html[Virtio_user as Exception Path].
+
+In OpenShift Container Platform version 4.14 and later, you can use non-privileged pods to run DPDK applications alongside the tap CNI plugin. To enable this functionality, you need to mount the `vhost-net` device by setting the `needVhostNet` parameter to `true` within the `SriovNetworkNodePolicy` object.
+
+.DPDK and TAP example configuration
+image::348_OpenShift_rootless_DPDK_0923.png[DPDK and TAP plugin]
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You are logged in as a user with `cluster-admin` privileges.
+* Ensure that `setsebools container_use_devices=on` is set as root on all nodes.
++
+[NOTE]
+====
+Use the Machine Config Operator to set this SELinux boolean.
+====
+
+.Procedure
+
+. Create a file, such as `test-namespace.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-namespace
+  labels:
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+----
+
+. Apply the yaml by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f test-namespace.yaml
+----
+
+. Create the following `SriovNetworkNodePolicy` object, and then save the YAML in the `sriov-node-network-policy.yaml` file:
```

Comment:
In the other steps, this step is to create the YAML, not to create the object. It should be consistent. 

> . Create a file, such as `sriov-node-network-policy.yaml`, with content like the following example: